test/plugins/metrics.test.js

```diff
@@ -63,7 +62,11 @@ describe('request metrics plugin', function() {
                     assert.isAtLeast(metrics.latency, 150);
                     assert.isAtLeast(metrics.totalLatency, 150);
                     assert.equal(metrics.path, '/foo');
-                    assert.equal(metrics.connectionState, undefined);
+                    // close in Node >= 10
+                    assert.include(
+                        [undefined, 'close'],
```

Comment:
What does that `'[undefined, 'close']` mean?

Reply:
Any of them. Undefined is the unknown connection state basically.

Reply:
What are the cases when the connection state would be `'close'`? Shouldn't it be `'close'` only when the request was aborted? In this case it doesn't seem like the request would be aborted.